stdlib/gzip.pyi

```diff
@@ -15,8 +15,14 @@ _ReadBinaryMode: TypeAlias = Literal["r", "rb"]
 _WriteBinaryMode: TypeAlias = Literal["a", "ab", "w", "wb", "x", "xb"]
 _OpenTextMode: TypeAlias = Literal["rt", "at", "wt", "xt"]
 
-READ: Literal[1]
-WRITE: Literal[2]
+READ: int  # actually Literal[1] # undocumented
+WRITE: int  # actually Literal[2] # undocumented
```

Comment:
...But at the same time, I'd rather leave the ones that are _already in the stub_ as they are, unless they're causing a specific problem for somebody :)

```suggestion
READ: Literal[1]  # undocumented
WRITE: Literal[2]  # undocumented
```